### DISCUSSION TITLE
Some requests taking forever in beta

### DISCUSSION THREAD

**Discussion by AndreKR** [TomBursch/kitchenowl#527] [Q&A]
**Update:** Reverting to the `latest` tag has not solved the issue, I will investigate more.

-----

On some of my phones a beta version of Kitchen Owl gets installed. Those showed an error message that the server version is incompatible and I need to update the backend. So I changed the Docker tag from `latest` to `beta`.

Unfortunately now Kitchen Owl doesn't work anymore at all. Some requests take forever.

![image](https://github.com/user-attachments/assets/c2ff0229-7235-480b-b81d-f8064cf3be6e)

It's only some of the requests but I think it is always the same ones each time I reload, including the index, Icon-192.png, main.dart.js, facicon.ico, flutter_bottstrap.js and main.dart.js.

The server load is fine and `docker compose logs` shows only the requests, like
```
kitchenowl-front-1  | 172.19.0.1 - - [21/Oct/2024:17:08:00 +0000] "GET / HTTP/1.1" 200 2694 "-" ...
```
and about once a minute this line:
```
kitchenowl-back-1   | [pid: 31|app: 0|req: 60/60]  () {16 vars in 245 bytes} [Mon Oct 21 17:12:00 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 4 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 198)
```

For now I have reverted to `latest`. I did not restore the DB from backup, I hope that's ok.

**Comment by TomBursch** [comment]
Hmm strange, I've been using the beta version for the last month and haven't noticed any issues. Did you try clearing the page cache?

**Reply by AndreKR** [reply]
A Chrome restart (not just the tab) fixed it. 🤷 Sorry for the fuss.

**Reply by TomBursch** [reply]
No worries, glad it's working again 😄
